vignettes/custom-steps.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Custom steps"
+title: "Contributing: Writing custom steps"
```

Comment:
The overwhelming majority of users will use `add_code_step()`. This is really documentation for developers of _tic_, and should be framed as such.

Reply:
I changed the title and also added more information about the whole R6 argument passing.